app/routes.py

```diff
@@ -377,31 +378,46 @@ def remove_candidate():
 def missing_route():
     return render_template("404.html")
 
-
-# the update status function is the function to be ran repaeatedly hence the id
-# the function will be ran in 30 seconds intervals
-@scheduler.task('interval', id='do_update_status', seconds=10, misfire_grace_time=900)
-def update_status():
-    now = datetime.now()
+def schedule_job(id):
```

Comment:
Could you add this method to tools.py?  Let's keep routes.py strictly for routes

Reply:
Or better still, a different file for our jobs. maybe `jobs.py` or something